debug_icb.f90

```diff
@@ -37,3 +37,16 @@ subroutine debug_c(logfil_c, ndigit_c, mgetv0_c,
   mcgets = mcgets_c
   mceupd = mceupd_c
 end subroutine debug_c
+
+! Initialisation of the debug common block to "no debug".
+block data init
```

Comment:
can you move this into a separate new `.f90`? It should not be related to ICB and should happen regardless of users having ICB on or off. 

Currently, this source file is not part of the lib unless ICB is on.